cypress/integration/e2e/article.metaAndOphan.e2e.spec.js

```diff
@@ -0,0 +1,118 @@
+import { getPolyfill } from '../../lib/polyfill';
+import { disableCMP } from '../../lib/disableCMP.js';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+
+describe('E2E Page rendering', function () {
+	before(getPolyfill);
+
+	beforeEach(function () {
+		disableCMP();
+		setLocalBaseUrl();
+	});
+
+	const getExists = (selector) => cy.get(selector).should('have.length', 1);
+
+	describe('for WEB the document renders with the correct meta and analytics info', function () {
```

Comment:
These pieces of describe text with WEB and E2E don't really make sense anymore so you could just write whatever is useful in this context - not really a big deal, I guess it only helps when scrolling the results or looking at errors

Reply:
Actually, maybe not one, perhaps several, but ideally fewer!